spec/models/placement_spec.rb

```diff
@@ -37,4 +37,24 @@
     it { is_expected.to validate_presence_of(:school) }
     it { is_expected.to validate_presence_of(:status) }
   end
+
+  describe "scopes" do
+    describe "#order_by_subject_school_name" do
+      it "returns the placements ordered by their associated schools name" do
+        school_a = create(:placements_school, name: "Abbey School")
+        school_b = create(:placements_school, name: "Bentley School")
+
+        subject_a = create(:subject, name: "Art and design")
+        subject_b = create(:subject, name: "Biology")
+
+        placement_1 = create(:placement, school: school_b, subjects: [subject_a])
+        placement_2 = create(:placement, school: school_a, subjects: [subject_b])
+        placement_3 = create(:placement, school: school_b, subjects: [subject_b])
```

Comment:
Can you update this test to include a placement with _multiple_ subjects please?

This is the database query that I was worried about. Using a regular SQL join, I'd expect we'd get back duplicates of the same placement record, one for each subject. For example, Placement A with subjects Art and Biology could come back as two Placement A results – if that makes sense!

I think the change you made from `.joins(:subjects, :school)` to `.include(:subjects, :school)` _should have_ resolved this, but it'd be good to prove it with a test.

Reply:
Resolved in [3d9756d](https://github.com/DFE-Digital/itt-mentor-services/pull/531/commits/3d9756d9ceb6973fc9ea3d3d45afe6156d4d6e41)

Reply:
Thank you <3